engine/src/test/scala/cromwell/util/SampleWdl.scala

```diff
@@ -2306,7 +2306,8 @@ task shouldCompleteFast {
         |    Int a
         |    command {
         |        echo "The number was: ${a}"
-        |        sleep 0.2
+        |        # More than 0.15 so this should finish second
```

Comment:
This 0.1 second stuff still makes me kind of uneasy, wouldn't be surprised if Travis gets wonky with this